varname.py

```diff
@@ -471,6 +485,14 @@ def _get_node(caller: int, raise_exc: bool = True) -> Optional[ast.AST]:
         return None
 
     exet = executing.Source.executing(frame)
+    qualname = exet.code_qualname()
+    module = inspect.getmodule(frame)
```

Comment:
@alexmojaki 
Is there a better way for this with executing? Or this is already the best way.

Reply:
I think this is fine. The alternative is to compare `module.__file__` and `frame.f_code.co_filename` but I'm not actually sure if they always match.